### DISCUSSION TITLE
I made a tinygrad Iris classification tutorial for beginners.

### DISCUSSION THREAD

**Discussion by Skynse** [tinygrad/tinygrad#2148] [Show and tell]
Let me know if there are any issues with the tutorial, or any suggestions to improve it.

The tutorial explains a few of the common machine learning operations like zero_grad, opt.step, and loss.backward

https://dev.to/skynse/how-to-train-an-iris-dataset-classifier-with-tinygrad-1gee
